src/Nova/Post.php

```diff
@@ -28,6 +28,15 @@ class Post extends BaseResource
 
     public static $group = 'Website Blog';
 
+    public function actions(Request $request)
+    {
+        return [
+            (new Actions\PreviewPost($this->id))
+                ->onlyOnTableRow()
+                ->withoutConfirmation(),
```

Comment:
It is possible to require a confirmation before redirecting. The default behavior of a Nova actions would normally do so but I tried to mimic how WordPress directly links to a page without question.

Reply:
Let's have it open in a new tab. No confirmation needed.